src/components/select/select.test.jsx

```diff
@@ -320,6 +320,12 @@ describe('src/components/select/select', () => {
 
```

Comment:
We should probably do the same refactor on the above `'provided value sets the corresponding list item as selected'` test that we did for the custom field single choice input test of the same name.